### DISCUSSION TITLE
Pre-assembled S14 won't accept password for initial connect

### DISCUSSION THREAD

**Discussion by MaggieLeber** [simpleavr/s14clock#3] [Q&A] [answered]
The self-assembled s14 from Discussion #2 is working beautifully, and has been deployed to my office at work...

https://photos.app.goo.gl/siDzqW9D631TVXcj6

But the pre-assembled s14 from my second order starts up correctly but rejects the 12345678 password on the same phone I always use. 

Any ideas? IDs as Hardware 2.0, Firmware 2.01

**Comment by simpleavr** [comment, marked as answer]
Hi Maggie,

Apologies for the bug? and this did happened to another assembled unit / customer. I have not yet found out the reason for it, it has something in the AutoConnect libary. For the assembled units, I did a 48 hour burn-in and had previously log-in my home WiFi, and could have messed up w/ the memory / configuration.

And a complete erase and flash all will resolve this problem.

You have to flash the unit again. Please follow this link (wiki) for instructions.
https://github.com/simpleavr/s14clock/wiki#erase-and-flash-all
The binaries are inside the v2.01 release package,
https://github.com/simpleavr/s14clock/releases/tag/v2.01
you have to download bootloader.bin partitions.bin and firmware24_201.bin and use the web-based adafruit tool to flash your non-working unit.
Let me know if the problem presists.
Thanks for your purchase.

**Reply by MaggieLeber** [reply]
OK, I'll let you know how I make out. Minor annoyance but probably a learning experience.

**Reply by MaggieLeber** [reply]
All done. No big deal. Now working perfectly.  Thanks for your help!

**Comment by sbadger** [comment]
I also ran into this with the kit I purchased, but like you said it wasn't a big deal to correct it once I figured out that the baud rate was 921600.
